spec/support/authenticate.rb

```diff
@@ -0,0 +1,10 @@
+module Authenticate
+  def authenticate_user user
+    allow_any_instance_of(ApiController).to receive(:authenticate_api_user).and_return(true)
+    allow_any_instance_of(ApiController).to receive(:current_user).and_return(@user)
+  end
+end
+
+RSpec.configure do |config|
+  config.include Authenticate
```

Comment:
This could be moved to `rails_helper` right? It feels good to have all configuration in one place.